test/end-to-end/e2e-tools.js

```diff
@@ -197,3 +197,31 @@ module.exports.setIncidentFilter = function(filter) {
   }
   element.all(by.buttonText('Go')).first().click();
 };
+module.exports.createIncident = function(params) {
+  browser.get(browser.baseUrl + 'incidents');
+  element(by.buttonText('Create Incident')).click();
+  element(by.model('incident.title')).sendKeys(params.title ? params.title : 'blank');
+  element(by.model('incident.tags')).sendKeys(params.tags ? params.tags : '');
+  element(by.model('incident.locationName')).sendKeys(params.location ? params.location : '');
+  return element(by.buttonText('Submit')).click();
+};
+
+module.exports.addReportToIncident = function(reportParams, incidentParams) {
+  browser.get(browser.baseUrl + 'reports');
+  element.all(by.css('.addIdentifier')).first().click();
+  return element(by.cssContainingText('tr td', incidentParams.location)).click();
+};
+
+module.exports.filterByTag = function(tags) {
```

Comment:
I think we should have two independent functions here, `setIncidentFilter` and `getIncidentTitles`. `getIncidentTitles wouldn't need to take the `tags` argument.